dev/tests/integration/testsuite/Magento/Store/Model/StoreTest.php

```diff
@@ -22,6 +21,11 @@
  */
 class StoreTest extends \PHPUnit\Framework\TestCase
 {
+    /**
+     * Session ID in query param
+     */
+    const SESSION_ID_QUERY_PARAM = 'SID';
```

Comment:
Please explicitly specify visibility of this const. Also, now that SIDs and all their corresponding features are removed, I'm not sure if we should leave this param name as SESSION_ID_QUERY_PARAM with a SID value as that could be a bit misleading. In fact, the test that uses this only checks if a GET param is retained, so we can use any test param name.